secure_message/common/utilities.py

```diff
@@ -140,8 +147,9 @@ def add_business_details(messages):
 
 def add_users_and_business_details(messages):
     """Add both user and business details to messages based on data from party service"""
-    messages = add_to_and_from_details(messages)
-    logger.info("Sucessfully added to and from details")
+    messages = get_to_details(messages)
+    messages = get_from_details(messages)
```

Comment:
I think these methods would express their intent more clearly by using `add_` rather than `get_` as they add the details to the messages passed to them, rather than extract the 'to' and 'from' details.

Reply:
Agreed, changed them all